### PR TITLE
Wp 3

### PR DESCRIPTION
**Pull request by OfryL**

### CODE REVIEW

File: locomotion-sdk/src/Components/Drawer/index.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+const StyledDrawerLabel = styled.View`
+  margin: 0 35px;
+  border-color: #dfdfdf;
+  border-bottom-width: 1px;
+  padding: ${drawerPadding}px;
+  width: 100%;
+  flex-direction: row;
+  flex: 1;
+`;
```
Comment (oriefrati) [inline locomotion-sdk/src/Components/Drawer/index.js]: create styled file

File: locomotion-sdk/src/Components/PageHeader/index.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+`);
+
+const HeaderIcon = styled.Image(({ side }) => `
+  width: ${side === 'left' ? 19 : 13};
+  height: ${side === 'left' ? 19 : 13};
+`);
+
+const Header = styled.View`
+text-align: center;
+`;
```
Comment (oriefrati) [inline locomotion-sdk/src/Components/PageHeader/index.js]: create styled file

File: locomotion-sdk/src/pages/RideHistory/index.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React, { Fragment } from 'react';
+import Config from 'react-native-config';
+import PageHeader from '../../Components/PageHeader';
+
+const { CONTACT_US_URL: uri } = Config;
```
Comment (oriefrati) [inline locomotion-sdk/src/pages/RideHistory/index.js]: remove

File: server/lib/settings/index.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  return {
+    ...simpleSetting,
+    value: parsedValue,
+  };
+};
+
+const getAllSettingFromDb = async () => {
+  const foundSettings = await Setting.findAll({});
+  const settingsList = [];
+  foundSettings.forEach(foundSetting => {
```
Comment (oriefrati) [inline server/lib/settings/index.js]: replace to .map

File: server/web-ui/src/contexts/settingsContainer/api.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+    return settings;
+};
+// export const add = async values => networkService.post('/api/v1/admin/users', values);
+export const update = async (id, values) => (await networkService.patch(`/api/v1/admin/settings/${id}`, values)).data;
+// export const find = async (id) => {
+//     const { data: driver } = await networkService.get(`/api/v1/admin/users/${id}`);
+//     return driver;
+// };
+//
+// export const remove = async id => (await networkService.delete(`/api/v1/admin/users/${id}`)).data;
```
Comment (oriefrati) [inline server/web-ui/src/contexts/settingsContainer/api.js]: cleanup

File: server/web-ui/src/contexts/settingsContainer/index.js [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+const useSettings = () => {
+    const [settingsMap, setSettingsMap] = useState([]);
+    const [settingsObj, setSettingsObj] = useState({});
+
+    const loadSettings= async () => {
+        const newSettingsObj = {};
+        const SettingsData = await getSettings();
+        setSettingsMap(SettingsData);
+        SettingsData.forEach(setting => {
```
Comment (oriefrati) [inline server/web-ui/src/contexts/settingsContainer/index.js]: use map

**Review by OfryL — COMMENTED** [review]
👍

**Review by OfryL — COMMENTED** [review]
🥇

**Pull request merged**